src/content/docs/features/window-customization.mdx

```diff
@@ -2,13 +2,207 @@
 title: Window Customization
 ---
 
-import Stub from '@components/Stub.astro';
+Tauri provides lots of options for customizing the look and feel of your app's window. You can create custom titlebars, have transparent windows, enforce size constraints, and more.
 
-<Stub>
+## Configuration
 
-    - Based on https://tauri.app/v1/guides/features/window-customization
-    - Take
-    into account feedback in
-    https://github.com/tauri-apps/tauri-docs/issues/1344#issuecomment-1701912344
+There are three ways to change the window configuration:
 
-</Stub>
+- [Through tauri.conf.json](/references/v2/config/#windowconfig)
+- [Through the JS API](/references/v2/js/core/namespacewindow/#window)
+- [Through the Window in Rust](https://docs.rs/tauri/2.0.0-beta/tauri/window/struct.Window.html)
```

Comment:
It depends on the use case, but you don't have to stick with only one method. You can create a window by configuring `tauri.conf.json` and then use JS or Rust API to dynamically set up the window later at runtime.

Since I have read it so many times from 1.0 guide, it already feels normal to me 🤣  But I do agree it can be better,

Reply:
To make things easy, we can just slap a "external" icon `<Icon name="external" />` on each "Configuration" entry, so as not cause confusion with a TOC

Reply:
Ok, I just added it. 

`<svg>` in mdx were set `display: block`, so I add a custom class `.inline-icon` to custom.css and force the `<Icon>` here to use `display: inline-block`